SwrveSDKCommon/src/main/java/com/swrve/sdk/SwrveCommonEventsManager.java

```diff
@@ -0,0 +1,102 @@
+package com.swrve.sdk;
+
+import com.swrve.sdk.localstorage.ILocalStorage;
+import com.swrve.sdk.rest.IRESTClient;
+import com.swrve.sdk.rest.IRESTResponseListener;
+import com.swrve.sdk.rest.RESTResponse;
+
+import org.json.JSONException;
+
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+
+public class SwrveCommonEventsManager {
```

Comment:
Was this class formerly called SwrveEventsManager? Don't think we need the word common in there.